bin/rails

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+# This command will automatically be run when you run "rails" with Rails 4 gems installed from the root of your application.
+
+ENGINE_ROOT = File.expand_path('../..', __FILE__)
+ENGINE_PATH = File.expand_path('../../lib/copilot/engine', __FILE__)
```

Comment:
Let's try to make a checklist of all the things one must do to properly create a mountable engine. Ideally we can add it to process docs. Blog post?